static/src/stylesheets/module/facia-garnett/item-types/_fc-item--type-comment.scss

```diff
@@ -82,6 +86,7 @@
         &.fc-item--fluid-tablet {
             @include mq(tablet) {
                 .fc-item__container > .fc-item__meta {
+                    .fc-item__timestamp,
```

Comment:
This hides the duplicate timestamp on the specified facia 'comment' cards. As seen here https://trello.com/c/ahONx4Bs/250-brief-letters-tablet-up-to-wide-cards-show-date-stamp-twice